sipn_reanalysis_ingest/constants/cfsr.py

```diff
@@ -16,3 +16,5 @@
     ('monthly', 1): 'ds093.2',
     ('monthly', 2): 'ds094.2',
 }
+
+CFSR_DAILY_TAR_BEFORE = dt.date(2011, 4, 1) 
```

Comment:
We have the temporal relationship backwards. Tars are daily greater than or equal to this date. Could call it `CFSR_DAILY_TAR_ON_OR_AFTER` or `CFSR_5DAILY_TAR_BEFORE`?

Reply:
Done